web/package.json

```diff
@@ -65,8 +69,13 @@
     "@typescript-eslint/parser": "^5.10.1",
     "@vitejs/plugin-react": "^1.0.7",
     "autoprefixer": "^10.4.2",
-    "eslint": "^8.7.0",
-    "eslint-plugin-react": "^7.28.0",
+    "babel-eslint": "10.1.0",
+    "eslint": "8.32.0",
+    "eslint-plugin-react": "7.32.2",
+    "eslint-plugin-babel": "^5.3.1",
+    "eslint-plugin-import": "^2.27.5",
+    "eslint-plugin-simple-import-sort": "^10.0.0",
```

Comment:
We actually should avoid using ^ else might install package that might have bug or doesn't integrate properly if we do install instead of CI

But the newly added I have avoided ^ and left old one as it is

I see I missed two of them here will update them to without ^